plugins/LilyPad/Config.cpp

```diff
@@ -981,9 +983,12 @@ int LoadSettings(int force, wchar_t *file)
     for (int i = 0; i < sizeof(BoolOptionsInfo) / sizeof(BoolOptionsInfo[0]); i++) {
         config.bools[i] = GetPrivateProfileBool(L"General Settings", BoolOptionsInfo[i].name, BoolOptionsInfo[i].defaultValue, file);
     }
-    config.closeHacks = (u8)GetPrivateProfileIntW(L"General Settings", L"Close Hacks", 0, file);
-    if (config.closeHacks & 1)
-        config.closeHacks &= ~2;
+    config.closeHack = (u8)GetPrivateProfileIntW(L"General Settings", L"Close Hacks", 255, file);
+    if (config.closeHack == 1) {
+        config.closeHack = 0;
+    } else if (config.closeHack == 255) {
+        config.closeHack = (u8)GetPrivateProfileIntW(L"General Settings", L"Close Hack", 0, file);
+    }
```

Comment:
Might be better to not add old version compatibility code for an easily adjustable option - it may be more inconvenient from the user POV, but if you throw in compatibility code, who knows how long it'll stay there and that'll just make things less maintainable if this stuff builds up.